docs/examples/custom_root_field.json

```diff
@@ -1,11 +0,0 @@
-{
-  "definitions": {
-    "Pets": {
-      "title": "Pets",
-      "type": "array",
-      "items": {
-        "type": "string"
-      }
-    }
```

Comment:
This was almost completely duplicated in the python example already; it seemed simpler to just put it there in its entirety. Let me know if you prefer to keep this as a separate file though.

Reply:
yes, much better as one file.